R/execution.r

```diff
@@ -1,12 +1,14 @@
 #' @include options.R
 NULL
 
-displayenv = environment(publish_mimebundle)
+setClassUnion('recordedplotOrNULL', members = c('recordedplot', 'NULL'))
```

Comment:
@flying-sheep Can I suggest that functional changes to the code go in a different PR from the reformatting and style fixes? It's easier to review actual changes if they're not scattered through a big diff of trivial changes.